examples/jsm/capabilities/WebGPU.js

```diff
@@ -26,6 +23,12 @@ class WebGPU {
 
 	}
```

Comment:
the `WebGPU.isAvailable()` signature is changed, maybe do `return Boolean(isAvailable)` or create a new variable for the GPUAdapter

Reply:
hmm, I think creating a new variable is cleaner